packages/uniforms-antd/src/wrapField.js

```diff
@@ -39,6 +41,7 @@ export default function wrapField(
       colon={colon}
       hasFeedback
       help={showInlineError && error && errorMessage}
+      extra={extra || help}
```

Comment:
How about:

```diff
-help={showInlineError && error && errorMessage}
+help={help || showInlineError && error && errorMessage}
+extra={extra}
```

It'll allow users to decide indefinetely.

Reply:
I was hoping to preserve the same UI behavior across themes since help text on the bootstrap themes is always visible and I was trying to avoid having to edit all my SimpleSchema2s that pass a help property to uniforms.  If you prefer to stick close to antd's nomenclature, I'll live with it.

My only suggestion on your version is that the help message should come after the error message so that a static help message doesn't take precedence over a validation error message.
```diff
-help={showInlineError && error && errorMessage}
+help={(showInlineError && error && errorMessage) || help}
+extra={extra}
```

Reply:
I thought about it too, and for me, it's clear that AntD made the same decision:

> The prompt message. If not provided, the prompt message will be generated by the validation rule.

We can do the same (as in my previous diff). It won't be in align with Bootstrap, but it cannot be, as there are no two separate types of such prompts in Bootstrap.